core/src/main/java/nl/inl/blacklab/indexers/config/ConfigMetadataField.java

```diff
@@ -24,6 +24,9 @@
      * matching node.
      */
     private String forEachPath;
+    
+    /** Should this metadata field capture multiple values? */
+    private boolean isMultipleValues = false; 
```

Comment:
Waarom is het nodig om dit onderscheid te maken? Voor hoe BLS het veld in de response weergeeft? Of ook voor de indexering? Als alleen voor BLS, kunnen we dit dan niet tijdens indexeren detecteren? Dat maakt de indexconfig eenvoudiger.